docs/CLI.md

```diff
@@ -32,14 +33,58 @@ Let's go over the subcommands.
 
 Simply run `rudder list ...` providing the proper arguments.
 
-Example:
+The following example will list all event tables in database `RAW`, schema `mattermost_docs`:
+
 ```bash
 $ rudder list RAW mattermost_docs -a cca12345.us-east-1 -u joe@doe.com -r role_name -w warehouse_small
 Password: 
 DOC_EVENT
+NEW_TABLE
```

Comment:
needed?

Reply:
Just an example to show that it might return multiple rows